packages/components/image/src/image.tsx

```diff
@@ -29,6 +33,13 @@ const Image = forwardRef<"img", ImageProps>((props, ref) => {
   if (removeWrapper) {
     return img;
   }
+  if (isError || showFallback) {
+    return (
+      <div {...getErrorWrapperProps()}>
+        <img alt={img.props.alt} {...getErrorImageProps()} />
```

Comment:
1. don't set `alt={img.props.alt}` here. do it in `getErrorImageProps`.
2. use `<Component>` instead of `<img>`